internal/proto/controller/api/services/v1/scope_service.proto

```diff
@@ -99,6 +99,20 @@ service ScopeService {
       summary: "List all keys in a Scope."
     };
   }
+
+  // RotateKeys rotates and optionally rewraps all the keys found in the
+  // scope specified. If the scope is not found an error is returned. If
+  // the scope is empty, the global scope is used.
+  rpc RotateKeys(RotateKeysRequest) returns (RotateKeysResponse) {
+    option (google.api.http) = {
+      post: "/v1/scopes:rotate-keys"
+      body: "*"
+      // response_body: "item" // does this need to be here for an empty response?
```

Comment:
No, this is for when you want your response body to be unwrapped from the wrapper response message. It does not apply in this case.